documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -37,32 +44,50 @@ Examples of objects in Matlab include double-precision arrays, strings, cell arr
 
 In technical terms the function ``det`` is not a "method" of (i.e. function that works on) string objects.
 
-Matlab allows you to create your own objects, and methods to go with them, and this is what has been done with Horace. The objects are named d0d, d1d, d2d, d3d, d4d, and sqw, which correspond to 0D datasets, 1D datasets,..., 4D datasets, and general datasets of any dimensionality with full detector pixel information retained. The methods that apply to these objects are contained within the directories ``C:\\mprogs\\Horace\\@d0d`` etc. Methods that apply to different objects can have the same name, and in the interests of keeping things relatively intuitive there are several examples of this within Horace, e.g. if you type ``plot(w)`` then the object ``w`` will be correctly plotted (unless it is a d0d or d4d object, of course!) because there is a ``plot`` method in all of the d1d, d2d, d3d, and sqw directories.
+Matlab allows you to create your own objects, and methods to go with them, and this is what has been done with
+Horace. The objects are named d0d, d1d, d2d, d3d, d4d, and sqw, which correspond to 0D datasets, 1D datasets,..., 4D
+datasets, and general datasets of any dimensionality with full detector pixel information retained. The methods that
+apply to these objects are contained within the directories ``C:\\mprogs\\Horace\\@d0d`` etc. Methods that apply to
+different objects can have the same name, and in the interests of keeping things relatively intuitive there are several
```

Comment:
I think "in the interests of ... simple" just makes this more difficult to read, so delete please